lib/transmission-rss/feed.rb

```diff
@@ -25,6 +24,9 @@ def initialize(config = {})
         @config = {}
         @url = config.to_s
       end
+
+      @validate_cert = @config['validate_cert'].nil? || !!@config['validate_cert']
```

Comment:
I moved this outside of the case to ensure it is always set and `nil` doesn't need to be handled. It wasn't a big deal but I thought the boolean flags should be handled consistently.